js/lbank.js

```diff
@@ -435,15 +435,15 @@ module.exports = class lbank extends Exchange {
         const timestamp = this.safeInteger (order, 'create_time');
         // Limit Order Request Returns: Order Price
         // Market Order Returns: cny amount of market order
-        const price = this.safeNumber (order, 'price');
-        const amount = this.safeNumber (order, 'amount', 0.0);
-        const filled = this.safeNumber (order, 'deal_amount', 0.0);
-        const average = this.safeNumber (order, 'avg_price');
+        const price = this.safeString (order, 'price');
+        const amount = this.safeString (order, 'amount', 0.0);
```

Comment:
This is the situation I was talking on that other PR – when there's a default value (third argument), the type of it should match the expected return type of the safeMethod – in this case it should be a string, so the default fallback value has to be `'0.0'`. Or, maybe, even better – remove the 0.0 defaults from here entirely.